modules/network-observability-flows-format.adoc

```diff
@@ -1,404 +1,307 @@
-// Automatically generated by 'hack/asciidoc-flows-gen.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
+// Automatically generated by 'openshift-apidocs-gen'. Do not edit.
 :_mod-docs-content-type: REFERENCE
 [id="network-observability-flows-format_{context}"]
 = Network flows format reference
```

Comment:
@jotak I need to revert the ID to what it was in the docs, as I have it here, so it doesn't break links in the docs. Upstream, its `[id="flowmetric-flows-netobserv-io-v1alpha1"]
= FlowMetric [flows.netobserv.io/v1alpha1]`